### DISCUSSION TITLE
I want the ability to specify `line-spacing` instead of `line-height`

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#59] [Wants]
Currently if you want to add spacing between two lines, you need to use `line-height.` This property has the annoying effect of always adding an amount of spacing above and below the block of text.

I’d suggest something like `line-spacing` that would, instead, specify how much spacing should be added between lines only. No padding would be added at the bottom and top of the block of text.

See: https://webwewant.fyi/wants/5f033971446abbbbcfbdab4a/

**Comment by captainbrosset** [comment]
At first sight, this sounds really useful, and I can appreciate the frustration here, having been there. But I'm wondering if a `line-spacing` property wouldn't create inconsistent lines. `line-height` has the advantage of leading to lines that are always the same height. `line-spacing` would only insert space between line boxes, and therefore those lines boxes could be any height they wanted. Maybe both properties would be useful together, but it seems like `line-height` handles both things already.

**Comment by ephys** [comment]
Hello, I'm the one that filled in this request

It turns out that a CSS feature handling just this is being worked on: https://medium.com/microsoft-design/leading-trim-the-future-of-digital-typesetting-d082d84b202

It's not a line-spacing, that still gets covered by line-height, but it covers the same use cases

**Comment by captainbrosset** [comment]
I just sent a link to this issue to fantasai, who works on this proposal: https://twitter.com/patrickbrosset/status/1296349099688636420

**Comment by aarongustafson** [comment]
I’m going to add this one to the site even though a fix is inbound because it may help others find the solution.
